.vscode/settings.json

```diff
@@ -2,5 +2,6 @@
   "mochaExplorer.require": "ts-node/register",
   "mochaExplorer.files": "**/test/**/*.test.ts",
   "mochaExplorer.cwd": "./",
-  "typescript.tsdk": "node_modules/typescript/lib"
+  "typescript.tsdk": "node_modules/typescript/lib",
+  "jest.jestCommandLine": "packages/integration-test/node_modules/.bin/jest"
```

Comment:
```suggestion
  "typescript.tsdk": "node_modules/typescript/lib"
```

Reply:
without this jest.jestCommandLine the jest extension in vscode doesn't find jest binary.

Reply:
my problem is that you're setting a global setting to the monorepo to debug a specific package

Reply:
So I will put it in the .vscode of this specific package, thank you for the review.